### ISSUE TITLE
The label gets out of bounds in example

### ISSUE THREAD

**Issue by gerchicov-bp** [intuit/CardParts#19]
iPhone 5s simulator:
![simulator screen shot - iphone 5s - 2018-05-29 at 15 13 49](https://user-images.githubusercontent.com/11498205/40658179-0cf0134a-6353-11e8-809b-15daaed15d12.png)

**Comment by richardtop** [comment]
I can confirm the issue on iPhone SE.

**Comment by lwdupont** [comment]
@croossin Is this an issue in the CardParts code, or just the example? Can I take a poke around at it?

**Comment by croossin** [comment]
@lwdupont That would be awesome if you could!

**Comment by lwdupont** [comment]
@croossin So I looked at it. 

Thinking CardParts should handle this, I traced into where it's setting up the CardPartsTitleView. I tweaked updateContraints() slightly to pin the horizontal to both edges. I'll open up a pull request soonish, maybe tomorrow. 

Results:

![iphone 8 plus](https://user-images.githubusercontent.com/50843/40759789-7030a05a-6448-11e8-9f55-97d26fe97e6d.png)
![iphone se](https://user-images.githubusercontent.com/50843/40759790-704eea1a-6448-11e8-895c-c100f0a9ac36.png)
![iphone x](https://user-images.githubusercontent.com/50843/40759791-706c0dde-6448-11e8-9c2c-4fc47e0c061f.png)
